modules/swagger-codegen/src/main/resources/Java/libraries/feign/build.gradle.mustache

```diff
@@ -109,7 +109,7 @@ dependencies {
     compile "com.fasterxml.jackson.core:jackson-core:$jackson_version"
     compile "com.fasterxml.jackson.core:jackson-annotations:$jackson_version"
     compile "com.fasterxml.jackson.core:jackson-databind:$jackson_version"
-    compile "com.fasterxml.jackson.datatype:jackson-datatype-{{^java8}}joda{{/java8}}{{#java8}}jsr310{{/java8}}:$jackson_version"
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-{{#joda}}joda{{/joda}}{{#java8}}jsr310{{/java8}}:$jackson_version"
```

Comment:
Commit ddee1b5 always sets a `dateLibrary` in `additionalProperties` and `dateLibrary` is initialized to "joda" so this can never happen or am I missing somthing here?